include/udp_handler.h

```diff
@@ -18,6 +18,10 @@
 #include "lwip/sys.h"
 #include <lwip/netdb.h>
 
+#ifdef __cplusplus
```

Comment:
Why is this needed ? I don't want to use this with C++

Reply:
yes we are not using it but if someone add this repo in their project, the compiler will know that this file is C not C++ and will compile it with C compiler instead of C++ which add random numbers at the end of variable name for polymorphism and linker will fail.

Reply:
Oh, sorry, I didn't see the commit message. But have you tested this ? Like using this in a .cpp esp-app ?

Reply:
Yes I am working on another project where I used it and yes it is working.

Reply:
Cool

Reply:
Yes I am working on a flight controller and wanted to add MAVLINK support or atleast logging over wifi. For now it is just an IMU code and still building. https://github.com/shivam5594/ESP32_MPU9250

Reply:
Nice, it uses a cam board. If you need any contributor let me know ;)